lib/koans/10_map_sets.ex

```diff
@@ -5,19 +5,27 @@ defmodule MapSets do
 
   @set MapSet.new([1, 2, 3, 4, 5])
 
-  koan "I am merely another collection but, you can perform some operations on me" do
-    new_set = MapSet.new(@set, fn x -> 3 * x end)
-
-    assert MapSet.member?(new_set, 15) == ___
-    assert MapSet.member?(new_set, 1) == ___
-  end
+  koan "I am very similar to a list" do
+    assert Enum.fetch(@set, 0) == {:ok, ___}
```

Comment:
Might be nice to include an example here of fetching from the list _and_ the map.

```elixir
koan "Iamvery similar to a list" do
  list = [1,2,3]
  set = MapSet.new(list)

  assert Enum.fetch(list, 1) == 2
  assert Enum.fetch(set, 1) == 2
end
```